manager/base.py

```diff
@@ -242,16 +242,11 @@ def vmaas_call(cls, endpoint, data):
         headers = {'Content-type': 'application/json',
```

Comment:
this headers could be maybe included to every POST request in vmaas_post_request function automatically and not needed to pass using argument, we communicate only with VMaaS using JSON anyway